networkx/algorithms/tests/test_planarity.py

```diff
@@ -461,12 +507,24 @@ def test_successful_face_traversal(self):
         assert face == [1, 2]
 
     def test_unsuccessful_face_traversal(self):
-        embedding = nx.PlanarEmbedding()
-        embedding.add_edge(1, 2, ccw=2, cw=3)
-        embedding.add_edge(2, 1, ccw=1, cw=3)
+        embedding = nx.PlanarEmbedding(
+            {1: {2: {"cw": 3, "ccw": 2}}, 2: {1: {"cw": 3, "ccw": 1}}}
+        )
         with pytest.raises(nx.NetworkXException):
             embedding.traverse_face(1, 2)
 
+    def test_forbidden_methods(self):
+        embedding = nx.PlanarEmbedding()
+        #  embedding = nx.Graph([(0, 1), (1, 2), (3, 4), (4, 5)])
```

Comment:
Should we remove this comment that doesn't seem to be relevant anymore?

Reply:
done